packages/node_modules/nav-frontend-skjema/src/textarea.tsx

```diff
@@ -107,7 +107,7 @@ class Textarea extends React.Component<TextareaProps> {
     };
 
     static defaultProps = {
-        maxLength: 2000,
+        maxLength: undefined,
```

Comment:
Tenker kanskje denne defaulten bare kan fortsette å være `2000`. Og så endrer vi bare kommentaren til `maxLength` i interface til: 

> Maks antall tegn som kan skrives inn i tekstområdet. Angi `0` for å skru av tegn-begrensningen. Default er `2000`.

Da slipper de som allerede bruker denne komponenten å oppdatere noe. Retter du opp og legger til et eksempel med bruk av `maxLength={0}` for å skru av tegnbegrensning i `*.overview.mdx`-fila?

Tror også vi bør fikse slik at `<p className="textarea--medMeta__teller">`-elementet på [linje 184](https://github.com/navikt/nav-frontend-moduler/blob/2b040a94cf65bf413d4e5886b40539b9a2c51660/packages/node_modules/nav-frontend-skjema/src/textarea.tsx#L184) blir fjernet om bruker vil skru av tegnbegrensningen.